dss-api.yml

```diff
@@ -126,6 +126,73 @@ definitions:
       - uuid
       - name
       - versions
+  blob_common_metadata:
+    type: object
+    description: Common fields between the files list of a bundle and the metadata tracking an unique file.
+    properties:
+      uuid:
+        type: string
+        description: File unique ID
+      timestamp:
+        type: integer
+        format: int64
+        description: Timestamp of file creation measured in milliseconds since epoch
```

Comment:
ISO timestamps are restricted to second-level accuracy.  We should ideally have more accuracy than that.

Having said that, it seems people do tack on the milliseconds onto ISO timestamps, though at that point they're not ISO 8601 compliant.

Thoughts?